packages/react/src/components/actionable-item/ActionableItem.tsx

```diff
@@ -35,44 +38,38 @@ export interface IActionableItemProps {
     dropProps?: Partial<IDropOwnProps>;
 }
 
-export class ActionableItem extends Component<IActionableItemProps & HTMLAttributes<HTMLDivElement>> {
-    static defaultProps: Partial<IActionableItemProps> = {
-        actions: [],
-    };
-
-    render() {
-        const actionableItemClasses: string = classNames(
-            {'cursor-pointer': !!this.props.onItemClick},
-            'actionable-item-content actionable-item-container align-middle inline-block mod-border',
-            this.props.containerClassName
-        );
-        const dropPodClasses: string =
-            'actionable-item-container actionable-item-dots align-middle cursor-pointer inline-block mod-border-top mod-border-right mod-border-bottom';
-
-        return (
-            <div {..._.omit(this.props, 'actions', 'onItemClick', 'dropProps', 'containerClassName')}>
-                <div
-                    className={actionableItemClasses}
-                    onClick={(e: MouseEvent<HTMLDivElement>) => this.props.onItemClick?.(e)}
-                >
-                    {this.props.children}
-                </div>
-                {this.props.actions && this.props.actions.length ? (
-                    <Drop
-                        id={this.props.id}
-                        positions={[DropPodPosition.bottom, DropPodPosition.top]}
-                        buttonContainerProps={{className: 'inline-block'}}
-                        renderOpenButton={(onClick: () => void) => (
-                            <div onClick={onClick} className={dropPodClasses}>
-                                <Svg svgName="moreAppend" svgClass="icon mod-12 flex" />
-                            </div>
-                        )}
-                        {...(this.props.dropProps || {})}
-                    >
-                        <ListBox items={this.props.actions} noActive />
-                    </Drop>
-                ) : null}
-            </div>
-        );
-    }
-}
+export const ActionableItem: FunctionComponent<IActionableItemProps> = ({
+    id,
+    className,
+    containerClassName,
+    actions = [],
+    onItemClick,
+    dropProps,
+    children,
+}) => (
+    <div
+        className={classNames('actionable-item inline-flex mod-border mod-rounded-border-2 overflow-hidden', className)}
+    >
```

Comment:
We previously allowed HTML attributes on the div, it would be a breaking change not to allow it anymore

Reply:
I would target next